dev-libs/liblzw/liblzw-0.2-r1.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="small C library for reading LZW compressed files (.Z)"
+HOMEPAGE="https://github.com/vapier/liblzw"
+SRC_URI="mirror://sourceforge/freestdf/${P}.tar.lzma"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~hppa ~ia64 ~m68k ~s390 ~sh ~x86"
+
+src_configure(){
+	econf --enable-static=no
```

Comment:
Wouldn't `--disable-static` work?

Reply:
You're right.
Actually i've tried it with disable-static first but got an ```unrecognized option``` first. Checked it again and it works.. Probably had a typo and didn't looked carefully enough to see it.